### PR TITLE
[release] update changelog for 0.2.119

### PR DESCRIPTION
**Pull request by chrisvittal**

### CODE REVIEW

**Comment by danking** [comment]
For the record, until we have a fix for GRCh38 VEP image building, we have to manually copy the 0.2.118 tag to the 0.2.119 tag for vep-grch38-95.
```
skopeo copy docker://us-docker.pkg.dev/hail-vdc/hail/hailgenetics/vep-grch38-95:0.2.118 docker://us-docker.pkg.dev/hail-vdc/hail/hailgenetics/vep-grch38-95:0.2.119                                                      
```

I have done this and retried the ci-test.

**Comment by chrisvittal** [comment]
It passed.

File: hail/python/hail/docs/change_log.md [hunk 1]
```diff
@@ -33,6 +33,27 @@ Please note that **forward compatibility should not be expected, especially
 relating to file formats**: this means that it may not be possible to use
 an earlier version of Hail to read files written in a later version.
 
+## Version 0.2.119
+
+### New Features
+- (hail#12081) Hail now uses [Zstandard](https://facebook.github.io/zstd/) as
+  the default compression algoritm for table and matrix table storage. Reducing
```
Comment (danking) [inline hail/python/hail/docs/change_log.md:40]:
```suggestion
  the default compression algorithm for table and matrix table storage. Reducing
```

**Review by danking — CHANGES_REQUESTED** [review]
LGTM after this

File: hail/python/hail/docs/change_log.md [hunk 2]
```diff
@@ -33,6 +33,27 @@ Please note that **forward compatibility should not be expected, especially
...
+  file size around 20% in most cases.
```
Comment (danking) [inline hail/python/hail/docs/change_log.md:41]: I think we need a bigger notice of the change of file format. Can we add a new section ### File Format with a message like "The file format is now X.Y.Z. Previous versions of Hail will not be able to read files written by this version of Hail."

**Review by danking — CHANGES_REQUESTED** [review]

**Review by danking — APPROVED** [review]

**Pull request merged**